src/main/java/seedu/duke/commands/project/ProjectSelectCommand.java

```diff
@@ -11,14 +11,14 @@ public ProjectSelectCommand(int itemIndex) {
         this.itemIndex = itemIndex;
     }
 
-    public void executeCommand(ProjectList projects) {
+    public String executeCommand(ProjectList projects) {
         try {
-            projects.selectProject(itemIndex);
+            return projects.selectProject(itemIndex);
         } catch (NumberFormatException | IndexOutOfBoundsException | NullPointerException e) {
             if (projects.getProjectList().size() == 0) {
-                System.out.println("I am empty!!!");
+                return "I am empty!!!";
             } else {
-                System.out.println("Invalid project ID");
+                return "Invalid project ID";
```

Comment:
Similar case here, would extracting the printing function to UI class be neater?